src/core/dbus-service.c

```diff
@@ -199,6 +199,39 @@ static int bus_service_set_transient_property(
 
                 return 1;
 
+        } else if (streq(name, "FileDescriptorStoreMax")) {
+                uint32_t u;
+
+                r = sd_bus_message_read(message, "u", &u);
+                if (r < 0)
+                        return r;
+
+                if (mode != UNIT_CHECK) {
+                        s->n_fd_store_max = u;
+                        unit_write_drop_in_private_format(UNIT(s), mode, name, "FileDescriptorStoreMax=%u", (unsigned) u);
```

Comment:
the reason was that `s->n_fd_store_max` is actually of type `unsigned`, not uint32_t, the only reason we have it as uint32_t is that i comes from dbus... But maybe that's a poor reason, dunno ;-)